DFC.ServiceTaxonomy.Neo4j/Types/Relationship.cs

```diff
@@ -38,5 +39,12 @@ public List<string> ValidationErrors
                 return errors;
             }
         }
+
+        public override string ToString()
+        {
+            return $@"[:{RelationshipType}]->(:{string.Join(":", DestinationNodeLabels)})
+{DestinationNodeIdPropertyName}:
+{string.Join(Environment.NewLine, DestinationNodeIdPropertyValues)}";
+        }
```

Comment:
Formatting looks a little weird here, or is it GitHub?

Reply:
it's a `@` string which allows new lines.
It's like this to get the formatting right in the log files.